server/plugin.go

```diff
@@ -132,6 +154,9 @@ func (p *Plugin) GetClientForApp() msteams.Client {
 
 func (p *Plugin) GetURL() string {
 	config := p.API.GetConfig()
+	if config.ServiceSettings.SiteURL == nil {
```

Comment:
The system was already panicking if there was no SiteURL in the config. Probably we should make it somehow better.